browser.d.ts

```diff
@@ -1,5 +1,10 @@
+/// <reference lib="dom"/>
 import * as core from './core';
 
+export type FileTypeResult = core.FileTypeResult;
```

Comment:
This can be written as well like:
```ts
export {FileTypeResult, FileExtension, MimeType} from './core';
```

Reply:
I thought so too, but I'm not 100% sure since these are TS types. TS has a lot of bugs, so I went with the simplest pragmatic solution.

Reply:
Sure no problem.

I should have been more careful writing the TypeScript.